lz-string/index.d.ts

```diff
@@ -1,4 +1,4 @@
-﻿// Type definitions for lz-string v1.3.3
+﻿// Type definitions for lz-string v1.3.4
```

Comment:
The compressToEncodedURIComponent and decompressFromEncodedURIComponent methods appeared in lz-string 1.3.5.